frontend/public/components/overview/pods-overview.tsx

```diff
@@ -179,7 +179,10 @@ export const PodsOverviewContent: React.SFC<PodsOverviewContentProps> = ({
       variant="info"
       title={t('public~Waiting for the build')}
       actionLinks={
-        <AlertActionLink onClick={() => setShowWaitingPods(!showWaitingPods)}>
+        <AlertActionLink
+          onClick={() => setShowWaitingPods(!showWaitingPods)}
+          data-test="waiting-pods"
+        >
```

Comment:
not sure if this is consumed as I see it 

https://github.com/openshift/console/pull/12272/files#diff-982e6cfb636ca8ca63999882c646715710f983c0663a05dfd1e46c17e7714075R187-R190